src/redisearch_rs/trie_bencher/src/bencher.rs

```diff
@@ -162,6 +162,38 @@ impl OperationBencher {
         load_c_benchmark(&mut group, &self.keys);
         group.finish();
     }
+
+    /// Benchmark the find prefixes iterator.
+    ///
+    /// The benchmark group will be marked with the given label.
+    pub fn find_prefixes_group(&self, c: &mut Criterion, target: &str, label: &str) {
```

Comment:
Just a comment: Not sure what is the strategy to do the benchmarking. I wonder if we should be using more than one key as a query to the `operations` to benchmark. 

Ignore my comment if this has been the case for other operations.

Reply:
We generally try to exercise all the different "modes" of the operation.
If you look at the `operations.rs` file, you can see how we classify inputs based on what part of the algorithm they "hit".

For prefixes, we could add inputs that exit earlier as an additional data point.
Alternatively, we can add a dedicated corpus with a higher opportunity for matches, but it may be overkill.

Reply:
I've added another bench in 58a4161d6 using the full Gutenberg dataset, which is ~10x the size.
We can see that the timing scales (roughly) with the depth of the traversal, which is what we expect.